src/wmtk/Mesh.hpp

```diff
@@ -57,6 +59,8 @@ class TupleTag;
 }
 } // namespace multimesh
 
+class SimplicialComplex;
```

Comment:
we're trying to deprecate this class - why is it being added to more things?

Reply:
Vertexsmoothing is using simplicial complex so I just use it to the new function to adopt it. Can I add an issue and change it to simplexcollection later?